src/containers/Header/test/Header.test.js

```diff
@@ -97,16 +97,16 @@ describe('Header component', () => {
       wrapper.unmount()
     })
 
-    it('redirect on sidechain input works', () => {
+    it('redirect on custom network input works', () => {
       const wrapper = createWrapper()
 
-      const sidechainInput = wrapper.find('[className="sidechain_input"]')
-      sidechainInput.prop('onKeyDown')({
+      const customInput = wrapper.find('[className="custom_network_input"]')
+      customInput.prop('onKeyDown')({
         key: 'Enter',
-        currentTarget: { value: 'sidechain_url' },
+        currentTarget: { value: 'custom_url' },
```

Comment:
Ditto
```suggestion
        currentTarget: { value: 'custom_network_url' },
```

Reply:
This is just a placeholder for the test, so it doesn't matter what the value is